src/main/java/com/atlassian/db/replica/api/LsnReplicaConsistency.java

```diff
@@ -0,0 +1,86 @@
+package com.atlassian.db.replica.api;
+
+import com.atlassian.db.replica.internal.MonotonicMemoryCache;
+import com.atlassian.db.replica.spi.Cache;
+import com.atlassian.db.replica.spi.ReplicaConsistency;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.function.Supplier;
+
+public final class LsnReplicaConsistency implements ReplicaConsistency {
```

Comment:
Why do we need this class? We have `AuroraPostgresLsnReplicaConsistency ` already...